### DISCUSSION TITLE
How to correctly setup scheduling of the workflow

### DISCUSSION THREAD

**Discussion by wesraynbirdtilbury** [philosowaffle/peloton-to-garmin#427] [General]
The instructions say: _To have the action run on a schedule set the cron line in the ./github/workflows/sync_peloton_to_garmin.yml file._ - can someone please explain to me what the `cron` line of the file should look like? And how I setup a scheduled workflow on GitHib? Thank you!!!

(This is an amazing tool!)

**Comment by philosowaffle** [comment]
[This line](https://github.com/wjrt17peak/peloton-to-garmin/blob/master/.github/workflows/sync_peloton_to_garmin.yml#L13) defines the cron schedule.  If you do not make any changes, the default configuration will have the job run once a day around 1am.  [Here is a handy tool](https://crontab.guru/#0_1_*_*_*) for playing with and understanding cron scheduling.
